README.md

```diff
@@ -18,15 +18,23 @@ Engage with the community and get technical support on [the ODK-X forum](https:/
 
 ## Build 
 
+to build a docker image directly from the remote repository:
+
 `docker build --pull -t odk/sync-web-ui https://github.com/opendatakit/sync-endpoint-web-ui.git`
 
+alternatively, if you've cloned the repository to your local computer, nagivate to the directory containing the DockerFile, and use
+
+`docker build -t odk/sync-web-ui .`
+
 ## Run
 
-Use [RUN.md](RUN.md).
+*Note that the ODK Hamster Service is outdated and may no longer be supported, you are encouraged to set up your local ODK sync-endpoint with [sync-endpoint-default-setup](https://github.com/opendatakit/sync-endpoint-default-setup) (to customize your local setup, please look into ``docker-compose.yml`` inside the ``sync-endpoint-default-setup`` repository for more details).*
+
+Option 1: Use [RUN.md](RUN.md).
```

Comment:
It would be better to get rid of Run.md and bring all the instructions into the readme to be consistent with the other repos.

Reply:
should we keep the (outdated) instructions from RUN.MD in ReadMe, or should we simply delete everything in RUN.MD?

Reply:
delete everything in run.md and and consolidate to a single instruction set in readme